src/index.ts

```diff
@@ -51,7 +51,7 @@ export function ViteRsw(userOptions: RswPluginOptions): Plugin {
           });
 
           // fix: fetch or URL
-          code = loadWasm(code, path.basename(fileId), _name);
+          code = loadWasm(code, path.basename(fileId), config.base + _name);
```

Comment:
We can't assume the base path to be `/`. It could be anything. You can try with `vite build --base "/custom/base/path"`

Reply:
I try to run this command, thank you.

Reply:
After the build `import.meta.url` will be replaced with `__import_meta_url`

```js
// source code
code.replace(`new URL('${oPath}', import.meta.url)`, `new URL('${nPath}', location.origin)`)
// build
e.replace(`new URL('${o}', __import_meta_url)`,`new URL('${n}', location.origin)`)
```
<img width="939" alt="Screen Shot 2021-04-14 at 12 46 54 AM" src="https://user-images.githubusercontent.com/16164244/114590874-dae21880-9cbb-11eb-92b2-4a3e5a576b28.png">

Reply:
Yes, those are the changes from the latest `tsup` release (4.8.21). Not sure how we can fix this on our side except sticking with 4.8.20 for now.

Reply:
After the build `import.meta.url` will be replaced with `__import_meta_url`

```js
// source code
code.replace(`new URL('${oPath}', import.meta.url)`, `new URL('${nPath}', location.origin)`)
// build
e.replace(`new URL('${o}', __import_meta_url)`,`new URL('${n}', location.origin)`)
```
<img width="939" alt="Screen Shot 2021-04-14 at 12 46 54 AM" src="https://user-images.githubusercontent.com/16164244/114590874-dae21880-9cbb-11eb-92b2-4a3e5a576b28.png">

----
solution: RegExp

```js
// utils.ts

// ...
code = code.replace(/import\.meta\.url\.replace\(\/\\\.js\$\/, \'_bg\.wasm\'\);/, `fetch('${nPath}')`);
code = code.replace(`new URL('${oPath}',`, `new URL('${nPath}',`);
code = code.replace(/, import\.meta\.url\)/, `, location.origin)`);
```